src/mgr/ActivePyModules.cc

```diff
@@ -38,13 +38,17 @@
 #include "PyModuleRegistry.h"
 #include "PyUtil.h"
 
+#include "common/perf_counters_key.h"
+
 #define dout_context g_ceph_context
 #define dout_subsys ceph_subsys_mgr
 #undef dout_prefix
 #define dout_prefix *_dout << "mgr " << __func__ << " "
 
 using std::pair;
 using std::string;
+using std::vector;
+using std::string_view;
```

Comment:
Nit: Don't add these -- it's better to spell out the type as already done in most places.
```suggestion
```

Reply:
removed